packages/desktop-cli/README.md

```diff
@@ -47,4 +47,4 @@ Convert your [Streamlit](https://streamlit.io/) application into a desktop app w
 8. Run `npm run serve` or `yarn serve` for preview.
    - This command is just a wrapper of `electron` command as you can see at the `"scripts"` field in the `package.json`. It launches Electron and starts the app with `./build/electron/main.js`, which is specified at the `"main"` field in the `package.json`.
 9. Run `npm run dist` or `yarn dist` for packaging.
-   - This command just calls [`electron-builder`](https://www.electron.build/) (see the `"scripts"` field in the `package.json`) and it simply bundles the `./build` directory created in the step above. If you want to customize the built app, e.g., setting the icon, follow its instruction.
+   - This command bundles the `./build` directory created in the step above to a single application file (output to `dist/your-os/stlite-example.app`). To customize the built app, e.g. setting the icon, follow the [`electron-builder`](https://www.electron.build/) instructions.
```

Comment:
> to a single application file (output to `dist/your-os/stlite-example.app`)

I think this point is incorrect because 
* `stlite-example.app` is only for macOS and for Windows it would be `.exe`.
* Different type files like `*.dmg` for macOS are also created in the `dist` directory.

What about something like "This command bundles ... into application files in the `./dist` directory." ?

Reply:
Good point, used your suggestion and clarified it.